### PR TITLE
Do not merge yet: Alpakka 1.0-M3

### PR DESCRIPTION
**Pull request by 2m**
Should be merged after Alpakka release is done and docs are in place.

### CODE REVIEW

File: blog/_posts/2019-03-01-alpakka-1.0-M3-released.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+Highlights of the new release are:
+
+* Elasticsearch connector now has support for the new Akka Stream context propagation feature
+* AWS DynamoDb, Google Cloud Pub/Sub gRPC, Kudu now use Akka Stream attributes to inject needed resources
+* Apache Solr, AWS Kinesis, Geode, Google FCM, HBase, HDFS, IronMq, MongoDb, OrientDb, XML connectors have been refactored to more closely follow Alpakka connector guidelines
+* Most of the connectors now support Scala 2.13.0-M5
+ 
+For detailed changes and credits take a look at the [release notes](https://doc.akka.io/docs/alpakka/current/release-notes/1.0-M3.html) in the Alpakka documentation.
+
```
Comment (helena) [inline blog/_posts/2019-03-01-alpakka-1.0-M3-released.md]: link doesn't exist, it will right?

Comment (2m) [reply blog/_posts/2019-03-01-alpakka-1.0-M3-released.md]: Yes, when the release is complete.

File: blog/_posts/2019-03-01-alpakka-1.0-M3-released.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+Dear hakkers,
+
+After a bit more than a month since Alpakka 1.0-M2, we are delighted to announce the third milestone release of Alpakka 1.0.
+
+Highlights of the new release are:
+
+* Elasticsearch connector now has support for the new Akka Stream context propagation feature
+* AWS DynamoDb, Google Cloud Pub/Sub gRPC, Kudu now use Akka Stream attributes to inject needed resources
+* Apache Solr, AWS Kinesis, Geode, Google FCM, HBase, HDFS, IronMq, MongoDb, OrientDb, XML connectors have been refactored to more closely follow Alpakka connector guidelines
+* Most of the connectors now support Scala 2.13.0-M5
```
Comment (helena) [inline blog/_posts/2019-03-01-alpakka-1.0-M3-released.md]: Should we list which do?

File: blog/_posts/2019-03-01-alpakka-1.0-M3-released.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+ 
+For detailed changes and credits take a look at the [release notes](https://doc.akka.io/docs/alpakka/current/release-notes/1.0-M3.html) in the Alpakka documentation.
+
+Even modules that have been changed to meet Alpakka 1.0 standards might get more API-breaking changes before Alpakka reaches 1.0. If you prefer adapting to these changes in one bigger step, keep the version you are using until Alpakka 1.0 is final.
```
Comment (helena) [inline blog/_posts/2019-03-01-alpakka-1.0-M3-released.md]:
```suggestion
Modules that have been changed to meet Alpakka 1.0 standards might get more API-breaking changes before Alpakka reaches 1.0. If you prefer adapting to these changes in one bigger step, keep the version you are using until Alpakka 1.0 is final.
```

File: blog/_posts/2019-03-01-alpakka-1.0-M3-released.md (hunk 2 above)
Comment (2m) [reply blog/_posts/2019-03-01-alpakka-1.0-M3-released.md]: Good point. I'll link to the GitHub project that tracks this.

**Pull request merged**